internal/services/cdn/cdn_frontdoor_origin_resource.go

```diff
@@ -133,9 +133,11 @@ func resourceCdnFrontDoorOrigin() *pluginsdk.Resource {
 							ValidateFunc: validation.StringInSlice([]string{
 								"blob",
 								"blob_secondary",
+								"Gateway",
```

Comment:
I have one question, is this supposed to be `Gateway` or `gateway`?

Reply:
I have answered my own question, it's `Gateway` per the [documentation](https://learn.microsoft.com/azure/private-link/private-endpoint-dns). 🙂